src/test/resources/json/WikiTest/testWikiMapping/registrations/ng9em/wikis/index.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This is actually the list of wikis that is loaded at the start of the test, after which I get the first wiki and look at it.  So, the file is needed with the test as is.  I could alter the test to just look at one wiki, if you'd like.